testing/distributions-integ-tests/src/integTest/groovy/org/gradle/DistributionIntegrationSpec.groovy

```diff
@@ -36,7 +36,84 @@ abstract class DistributionIntegrationSpec extends AbstractIntegrationSpec {
 
     protected static final THIRD_PARTY_LIB_COUNT = 144
 
-    @Shared String baseVersion = GradleVersion.current().baseVersion.version
+    @Shared
+    String baseVersion = GradleVersion.current().baseVersion.version
+
+    def coreLibsFileNames = [
+        "gradle-base-asm-",
```

Comment:
🎉  Kudos for making this list sorted.

I suggest adding a check in the test to verify that _it is_ sorted so it never gets unsorted accidentally.